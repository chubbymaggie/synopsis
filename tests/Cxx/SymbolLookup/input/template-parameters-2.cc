template <class T> class myarray { /* ... */ };
template <class T> func() {}
template <class K, class V, template<class T> class C = myarray>
class Map
{
  C<K> key;
  C<V> value;
};
