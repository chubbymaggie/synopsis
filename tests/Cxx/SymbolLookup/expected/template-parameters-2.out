Identifier : T
declared at line 1 in <input>
ClassSpec : myarray
declared at line 1 in <input>
Identifier : T
declared at line 2 in <input>
Declarator : func
declared at line 2 in <input>
Identifier : K
declared at line 3 in <input>
Identifier : V
declared at line 3 in <input>
Identifier : T
declared at line 3 in <input>
Identifier : C
declared at line 3 in <input>
Name : myarray
declared at line 1 in <input>
ClassSpec : Map
declared at line 3 in <input>
Declarator : key
declared at line 6 in <input>
Declarator : value
declared at line 7 in <input>
